TESTS/integration/COMMON/mbed_app.json

```diff
@@ -0,0 +1,279 @@
+            "target.network-default-interface-type"     : "WIFI",
+            "wizfi310.tx"                               : "D1",
+            "wizfi310.rx"                               : "D0",
+            "wizfi310.rts"                              : "PB_1",
+            "wizfi310.cts"                              : "PB_13",
+            "wizfi310.rx-buffer-size"                   : 512,
+            "wizfi310.stacksize"                        : 1536,
+            "wizfi310.event-queue-size"                 : 1024,
+            "wizfi310.provide-default"                  : true
+        },
+        "DISCO_L475VG_IOT01A": {
+            "target.macros_remove"                      : ["MBED_TICKLESS"],
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_add"                     : ["WIFI_ISM43362"],
+            "target.network-default-interface-type"     : "WIFI"
+        },
+        "DISCO_L496AG": {
+            "target.macros_remove"                      : ["MBED_TICKLESS"],
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_add"                     : ["WIFI_WIZFI310"],
+            "target.network-default-interface-type"     : "WIFI",
+            "wizfi310.tx"                               : "PB_6",
+            "wizfi310.rx"                               : "PG_10",
+            "wizfi310.rts"                              : "PG_12",
+            "wizfi310.cts"                              : "PG_11",
+            "wizfi310.rx-buffer-size"                   : 512,
+            "wizfi310.stacksize"                        : 1536,
+            "wizfi310.event-queue-size"                 : 1024,
+            "wizfi310.provide-default"                  : true
+        },
+        "DISCO_F413ZH": {
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_add"                     : ["WIFI_ISM43362"],
+            "target.network-default-interface-type"     : "WIFI"
+        },
+        "DISCO_F469NI": {
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_add"                     : ["WIFI_WIZFI310"],
+            "target.network-default-interface-type"     : "WIFI",
+            "wizfi310.tx"                               : "D1",
+            "wizfi310.rx"                               : "D0",
+            "wizfi310.rts"                              : "D4",
+            "wizfi310.cts"                              : "D2",
+            "wizfi310.rx-buffer-size"                   : 512,
+            "wizfi310.stacksize"                        : 1536,
+            "wizfi310.event-queue-size"                 : 1024,
+            "wizfi310.provide-default"                  : true
+        },
+        "DISCO_F746NG": {
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_remove"                  : ["SD"]
```

Comment:
I think we should remove configurations like this one as it's already defined in targets.json

Reply:
I agree, many platforms should already have all the configuration they need set up in targets.json.  If they don't we need to fix that.  Otherwise, the extra configurations handle things like shields.  I defer to Marcelo's judgement, but I'd say we keep configurations provided by shields (except maybe that interferes with plans for CI?)

Reply:
Shields might be an opportunity to work on toy with a more standardized dependency management solution.

Reply:
fixed